INSTRUCTION.md

```diff
@@ -0,0 +1,24 @@
+# How to validate the changes
+
+#Use this command to run Bootstrap Script
```

Comment:
Consider adding a brief explanation of what the bootstrap script does, so users understand its purpose before running it.